plugins/languages/treesitter/treesitter.nix

```
@@ -284,7 +289,7 @@ helpers.neovim-plugin.mkNeovimPlugin config {
       # NOTE: Upstream state that the parser MUST be at the beginning of runtimepath.
       # Otherwise the parsers from Neovim takes precedent, which may be incompatible with some queries.
       (optionalString (cfg.settings.parser_install_dir != null) ''
-        vim.opt.runtimepath:prepend("${cfg.settings.parser_install_dir}")
+        vim.opt.runtimepath:prepend("${helpers.toLuaObject cfg.settings.parser_install_dir}")
```

```suggestion
        vim.opt.runtimepath:prepend(${helpers.toLuaObject cfg.settings.parser_install_dir})
```

`toLuaObject` will handle the quotes